python/doc/bibliography.rst

```diff
@@ -193,7 +197,7 @@ Bibliography
 .. [jin2005] R. Jin, W. Chen, and A. Sudjianto. *An efficient algorithm for
     constructing optimal design of computer experiments.*
     Journal of Statistical Planning and Inference, 134 :268-287, 2005.
-    `pdf <https://openturns.github.io/openturns/papers/jin2005.pdf>`__
+    `pdf <https://openturns.github.io/openturns/papers/jones1993.pdf>`__
```

Comment:
wrong line ?

Reply:
It looks like this reference is for another topic, related to LHS sampling.